README.md

```diff
@@ -1,2 +1,141 @@
-# sagastandard
-One Standard to bring them all, and in the darkness bind them.
+# Javascript Saga-Standard style
+### One Standard to bring them all, and in the darkness bind them.
+
+Based on [feross/standard] and [Flet/semistandard] with a few additionnal opiniated style rules.
+
+## Install
+```bash
+npm install -g sagastandard
+```
+
+## Rules
+Importantly:
+- **1tbs**: check [one-true-brace-style](http://eslint.org/docs/rules/brace-style)
+- **curly**: always require braces
+- **max-len at 80**: except for urls.
+- **NO console.log**
+- **NO debugger;***
+- **semicolons** check [Flet/semistandard]
+- Check [feross/standard] for the rest of the rules.
+
+## Usage
+
+The easiest way to use JavaScript Saga-Standard Style to check your code is to install it globally as a Node command line program. To do so, simply run the following command in your terminal (flag `-g` installs `sagastandard` globally on your system, omit it if you want to install in the current working directory):
+
+```bash
```

Comment:
For me its nice to have a quick install command at the top for people who don't want to read and a lengthier one for people who are less versed in the npm ecosystem.